Command/DoctrineODMCommand.php

```diff
@@ -22,19 +22,14 @@ public function __construct(private ManagerRegistry $registry)
     }
 
     /** @param string $dmName */
-    public static function setApplicationDocumentManager(Application $application, $dmName)
+    public static function setApplicationDocumentManager(Application $application, $dmName): void
```

Comment:
```suggestion
    public static function setApplicationDocumentManager(Application $application, string $dmName): void
```

I'll check these native types in another PR as I guess we have some more we can add, so feel free to ignore this comment in this PR.